presto-parser/src/main/java/com/facebook/presto/sql/QueryUtil.java

```diff
@@ -181,6 +181,18 @@ public static Query simpleQuery(Select select, Relation from, Optional<Expressio
                 limit));
     }
 
+    public static Query singleValueQuery(String columnName, String value)
+    {
+        return query(new QuerySpecification(
```

Comment:
Pull this into a separate commit (at the beginning of the commit list) and make `visitExplain` use this.